setup.py

```diff
@@ -3,22 +3,17 @@
 
 from __future__ import with_statement
 from setuptools import setup, find_packages
-
+from cc_plugin_ugrid import __version__
```

Comment:
Interesting, I was following the pattern in https://github.com/ioos/cc-plugin-ncei and https://github.com/ioos/compliance-checker because I was getting this error when I had the ugrid plugin installed locally:
```
Could not load ugrid-1.0 = cc_plugin_ugrid.checker_10:UgridChecker10 : [Errno 2] No such file or directory: '/Users/bobfratantonio/Documents/Dev/virtenvs/compliance-checker/lib/python2.7/site-packages/cc_plugin_ugrid-0.0.1.dev0-py2.7.egg/VERSION'
```

I'm interested to see the versioneer approach if thats more robust

Reply:
@ocefpaf can you provide an example of versioneer or PR?

Reply:
Sure! See https://github.com/ioos/cc-plugin-ugrid/pull/9

The release workflow would be based on tags and that should simplify the deployment and reduce issues like those.

@Bobfrat the changes in #9 are mostly `versioneer` auto-generated stuff. The commands to get there are:

```shell
conda install versioneer
versioneer install
```

and follow the instructions from there. I usually have the `setup.py` configure to be `pep440` and use the `v` before the version number tag on GitHub (there won't be a `v` in the actual version from the Python side, only on the tag.).